README.md

```diff
@@ -319,11 +319,24 @@ oidcHandler := oidcgin.New(
 
 ### Custom error handler
```

Comment:
Can we add something to the readme about the breaking change as well?

Reply:
Fixed.